### PR TITLE
Added support OpenBSD, NetBSD, DragonFly OS

### PR DESCRIPTION
**Pull request by Martinfx**
Hello i bring support OpenBSD, NetBSD, DragonFly operating systems.
M.

### CODE REVIEW

File: examples/Makefile [hunk 1]
```diff
@@ -328,7 +328,7 @@ ifeq ($(PLATFORM),PLATFORM_DESKTOP)
     ifeq ($(PLATFORM_OS),BSD)
         # Libraries for FreeBSD, OpenBSD, NetBSD, DragonFly desktop compiling
         # NOTE: Required packages: mesa-libs
-        LDLIBS = -lraylib -lGL -lpthread -lm
+        LDLIBS = -lraylib -lGL -lpthread -lm -lglfw
```
Comment (raysan5) [inline examples/Makefile]: GLFW shouldn't be required, now is compiled together with raylib in module [rglfw](https://github.com/raysan5/raylib/blob/master/src/rglfw.c).

Comment (Martinfx) [reply examples/Makefile]: ok

**Comment by Martinfx** [comment]
@raysan5 Its this pull request ok ?

**Comment by a3f** [comment]
@Martinfx Have you tested this on OpenBSD? CPANtesters has yet to report me that the Perl wrapper managed to build on an OpenBSD system. They seem to all fail because they don't find `<sys/soundcard.h>`. see e.g. http://www.cpantesters.org/cpan/report/ba78f6ce-63af-11e8-9a1a-c5a44eb4fde3

The test matrix for the most recent Alien-raylib release is here:
http://matrix.cpantesters.org/?dist=Alien-raylib%200.019

**Comment by Martinfx** [comment]
@a3f No but this bug is in mini_al header and is need #include <soundcard.h> only

**Comment by mackron** [comment]
I was just looking at BSD in mini_al just the other day! :)

I will be improving BSD support soon - it's the next thing on my list. I will be adding support for audioio.h and sndio. I think I'll also be restricting OSS to FreeBSD. Any other suggestions?

**Comment by Martinfx** [comment]
@mackron  no :D

**Pull request merged**